openmetadata-ui/src/main/resources/ui/playwright/e2e/dataInsightApp.ts

```diff
@@ -115,7 +112,7 @@ setup(
               }),
         }
       )
-      .toBe('success');
+      .not.toEqual(/(running|queued)/);
 
```

Comment:
after this, i think we need to add a success check as well, else we won't know what the status of the pipeline

Reply:
We are checking for success in next line. After a page reload